src/main/resources/XCarouselCode/XCarouselSheet.xml

```diff
@@ -163,7 +163,7 @@
               #set ($itemClass = "item slide_${object.number}")
             #end
             &lt;div class="${itemClass}"&gt;
-              &lt;img src="$!{imageUrl}?height=$!{imageHeight}" alt=''&gt;
+              &lt;img src="$!{imageUrl}" alt="$!{object.getProperty('title').value}"&gt;
```

Comment:
Always use `$escapetool.xml` when printing user-entered content into html. In this case, the `alt` attribute value needs escaping.